packages/rocketchat-api/server/default/info.js

```diff
@@ -1,5 +1,17 @@
 RocketChat.API.default.addRoute('info', { authRequired: false }, {
 	get: function() {
-		return RocketChat.Info;
+		const user = this.getLoggedInUser();
+
+		if (user && RocketChat.authz.hasRole(user._id, 'admin')) {
+			return {
```

Comment:
Actually it's the inverse of my comment above. This line is correct for the default route and the return on this same file below (when not authenticated) which uses the `v1.success` is not correct for the default route.